app/src/main/java/org/thoughtcrime/securesms/home/search/GlobalSearchAdapterUtils.kt

```diff
@@ -66,7 +66,7 @@ fun ContentView.bindQuery(query: String, model: GlobalSearchAdapter.Model) {
             binding.searchResultSubtitle.isVisible = true
             binding.searchResultTitle.text = model.messageResult.conversationRecipient.getSearchName()
         }
-        is GroupConversation -> {
+        is LegacyGroupConversation -> {
```

Comment:
What about groupsv2?

Reply:
This part seems unfinished so I'll need to hook it up proper

Reply:
Right so `LegacyGroupConversation` can actually cover the new group so I've updated the naming